content/docs/onpremise/deploy.md

```diff
@@ -0,0 +1,75 @@
++++
+title = "Deployment On-premises"
+description = "Instructions to deploy Kubeflow on-premise cluster"
```

Comment:
Let's standardize to `on-premises` (with the final "s") wherever we use this term. 

There's a lot of debate about the term currently. In the US, many people say "on-premise" but outside the US that sounds odd to most people. A *premise* is an assumption, whereas *premises* refers to a building, location, or site. The Google developer's style guide, which we follow in the Kubeflow docs, recommends "on-premises".